src/elements/content-sidebar/activity-feed/activity-feed/stories/ActivityFeed.stories.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import * as React from 'react';
+
+import { IntlProvider } from 'react-intl';
+
+import ActivityFeed, { type ActivityFeedProps } from '../ActivityFeed';
+import {
+    TIME_STRING_SEPT_27_2017,
+    TIME_STRING_SEPT_28_2017,
+    currentUser,
+    comment,
+    replies,
+    repliesProps,
+} from '../../comment/stories/common';
+import { FEED_ITEM_TYPE_VERSION } from '../../../../../constants';
+
+import { type FeedItems, type Comment } from '../../../../../common/types/feed';
```

Comment:
nit: could combine to `import type {foo, bar}`; I'll implement this if there are other changes needed.